content/events/2023-2024/2023-09-12-git-workshop.md

```diff
@@ -10,6 +10,7 @@ poster_position: "center"
 short_description: "Do you still manage your code by making 17 versions of them on your computer? Come learn git! The tool all the cool kids are using to manage their projects and work with other programmers."
 start_time: "2:00 - 3:00 PM EDT"
 location: Seminar Room (HP 5345)
```

Comment:
Can you update the locations to now say "Recording"?

Reply:
Kay done.